database/dataGenerator.js

```diff
@@ -9,20 +9,20 @@ const getNumInRange = (lower, upper) => {
   return Math.floor(Math.random() * (upper - lower)) + lower;
 };
 
-// TODO: use more realistic data (add random ranges of dates)
+const randomDate = (start, end) => {
+  const startTime = start.getTime();
+  const endTime = end.getTime();
+  const randomTime = startTime + ((endTime - startTime) * Math.random());
+  return new Date(randomTime);
+};
+
 const getCalendar = () => {
   const calendar = [];
-  const months = [31, 28, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31];
-  for (let month of months) {
-    const days = [];
-    for (let i = 1; i <= month; i += 1) {
-      if (getNumInRange(1, 10) < 3) {
-        days.push(i);
-      }
-    }
-    calendar.push(days);
+  for (let i = 0; i < 50; i++) {
```

Comment:
Did eslint say anything about using ++ instead of += ?

Reply:
Good catch! I'll fix that